### PR TITLE
tests: net: Disable test for mps2_an385

### PR DESCRIPTION
**Pull request by jgl-meta**
This test seems to run out of memory for this platform. Temp. disabling it.

### CODE REVIEW

**Comment by jgl-meta** [comment]
@cfriedt

**Review by cfriedt — APPROVED** [review]

**Review by jukkar — APPROVED** [review]

**Comment by galak** [comment]
> @stephanosio - I know this is one of your preferred platforms for testing. Just wanted to keep you in the loop. If it's an OOM scenario, i wonder what caused it. Maybe some of the recent libc stuff?

We should open a bug for this.  The platform has 4M of memory, so it being OOM seems very odd.

**Comment by stephanosio** [comment]
This should only have been merged after an issue was opened.

**Comment by cfriedt** [comment]
We don't always have issues for hotfixes 🤷 

But yea, @jgl-meta - can you create a bug report for it? It should be root-caused with a proper fix before release for sure.

File: tests/net/socket/tls/testcase.yaml [hunk 1]
```diff
@@ -13,3 +13,4 @@ tests:
   net.socket.tls.preempt:
     extra_configs:
       - CONFIG_NET_TC_THREAD_PREEMPTIVE=y
+    platform_exclude: mps2_an385
```
Comment (stephanosio) [inline tests/net/socket/tls/testcase.yaml:16]: https://github.com/zephyrproject-rtos/zephyr/issues/57405

**Comment by stephanosio** [comment]
Any PR (hotfix or not) disabling a test due to a bug should have an associated issue, ideally with a `FIXME` comment describing why the test was disabled with a link to the issue.

**Pull request merged**